### DISCUSSION TITLE
Predictive ability (statistical analysis) of PBPK Models

### DISCUSSION THREAD

**Discussion by akim2000** [Open-Systems-Pharmacology/Forum#1577] [Questions & Problems]
I am wondering what methods are used today to evaluate the predictive ability of a PBPK model. I have seen the use of fold error (commonly 2-fold, but maybe even 1.5 and 1.33-fold) for evaluating PK parameters, but is the same used/enough for predicted concentration of drugs in the body? Are there any newer, more desirable methods of evaluating a PBPK model? And if a 2-fold error method is still used these days, then would that in itself be enough statistical analysis in a paper to publish your results, or are there other analyses that are also commonly performed on top of that?

**Comment by Yuri05** [comment]
It depends on the concrete use case. E.g. for DDI other criteria are better than 2-fold: https://pubmed.ncbi.nlm.nih.gov/21036951/

S. also e.g.

- https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6856026/
- https://pubmed.ncbi.nlm.nih.gov/31652029/
- https://www.oecd.org/chemicalsafety/risk-assessment/guidance-document-on-the-characterisation-validation-and-reporting-of-physiologically-based-kinetic-models-for-regulatory-purposes.pdf
- https://www.ema.europa.eu/en/documents/scientific-guideline/guideline-reporting-physiologically-based-pharmacokinetic-pbpk-modelling-simulation_en.pdf
